src/Core/Resolvers/SqlPaginationUtil.cs

```diff
@@ -292,7 +295,7 @@ RuntimeConfigProvider runtimeConfigProvider
                     if (backingColumnName is null)
                     {
                         throw new DataApiBuilderException(
-                            message: $"Cursor for Pagination Predicates is not well formed, {column.FieldName} is not valid.",
+                            message: $"Pagination token is not well formed because {column.FieldName} is not valid.",
```

Comment:
This should also be the exposed/mapped field name and not the backingColumnName in this error message.

Reply:
that's why we use the column.FieldName instead of the backingColumnName, this is safe.

Reply:
i see. the use of column / field together confused me

Reply:
Ya, the translation between exposed and backing columns/fields is a bit confuding, I thought about calling these NextLinkFields instead of columns, not sure if that's better or worse for readability.

Reply:
I think for consistency, now the we are movin everything to the expose = field, backing = column dichotomy, I will rename the class to NextLinkFields, the we can iterate through Fields and have Field.FieldName which should be easier to read